types/index.d.ts

```diff
@@ -371,6 +371,10 @@ declare namespace Realm {
         linkCredentials(credentials: Credentials): Promise<void>;
         callFunction(name: string, args: any[]): Promise<any>;
         refreshCustomData(): Promise<Object>;
+        push(serviceName: string): {
```

Comment:
@RedBeard0531 Going over this code when merging it with my changes (kh/v10-merged-types) I realize that we might want to have this `serviceName` be optional and provide a default name.